Sources/EdgeConstraints.swift

```diff
@@ -0,0 +1,27 @@
+//
+// Created by Alex Katsz on 2019-03-11.
+// Copyright (c) 2019 CariocaMenu. All rights reserved.
+//
+
+import UIKit
+
+public struct EdgeConstraints {
```

Comment:
Could you please add comments here for methods and properties ? I know some comment may seem useless in some cases, but I like to keep a 100% documentation coverage, as it's public code.
You can run jazzy to update the generated docs.

Reply:
Which jazzy version you are using? `0.9.5` complaining about `CariocaMenu+Protocols.swift#12`
```
"symbol_kind": "source.lang.swift.decl.typealias",
"warning": "undocumented"
```

Reply:
THanks! Just updated the docs. I'm using 0.9.5 too :)